Cannot build generate_parameter_library_example on RHEL 8

Hi,

We are trying to build MoveIT2 on RHEL 8 and we got problems with this generate_parameter_library. Can you help us to solve it and what is the minimun required Python version to build this? Our error message is following:

```--- stderr: generate_parameter_library_example
Traceback (most recent call last):
  File "/wrk1/ws_moveit2/install/generate_parameter_library_py/bin/generate_parameter_library_py", line 33, in <module>
    sys.exit(load_entry_point('generate-parameter-library-py==0.2.7', 'console_scripts', 'generate_parameter_library_py')())
  File "/wrk1/ws_moveit2/install/generate_parameter_library_py/bin/generate_parameter_library_py", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/local/lib/python3.6/site-packages/importlib_metadata/__init__.py", line 194, in load
    module = import_module(match.group('module'))
  File "/usr/lib64/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/wrk1/ws_moveit2/install/generate_parameter_library_py/lib/python3.6/site-packages/generate_parameter_library_py/main.py", line 419, in <module>
    class DeclareStruct:
  File "/wrk1/ws_moveit2/install/generate_parameter_library_py/lib/python3.6/site-packages/generate_parameter_library_py/main.py", line 421, in DeclareStruct
    def __init__(self, struct_name: str, fields: list[VariableDeclaration]):
TypeError: 'type' object is not subscriptable
gmake[2]: *** [CMakeFiles/admittance_controller_parameters.dir/build.make:75: admittance_controller_parameters/include/admittance_controller_parameters.hpp] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:181: CMakeFiles/admittance_controller_parameters.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
---
```

Command to build is: `PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/usr/local/lib64/pkgconfig colcon build --event-handlers desktop_notification- status- --cmake-args -DCMAKE_BUILD_TYPE=Release`

--- a/generate_parameter_library_py/generate_parameter_library_py/main.py
+++ b/generate_parameter_library_py/generate_parameter_library_py/main.py
@@ -987,7 +987,7 @@ class GenerateCode:
         yaml_file = sys.argv[2]
         with open(yaml_file) as f:
             try:
-                docs = yaml.load_all(f, Loader=yaml.FullLoader)
+                docs = yaml.load_all(f, Loader=yaml.Loader)
                 doc = list(docs)[0]
             except ParserError as e:
                 raise compile_error(str(e))

